browser/ui/toolbar/brave_location_bar_model_delegate.cc

```diff
@@ -6,27 +6,50 @@
 #include "brave/browser/ui/toolbar/brave_location_bar_model_delegate.h"
 
 #include "base/strings/utf_string_conversions.h"
+#include "brave/common/url_constants.h"
+#include "brave/components/brave_wallet/browser/buildflags/buildflags.h"
+
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
+#include "brave/common/extensions/extension_constants.h"
+#endif
+
 
 BraveLocationBarModelDelegate::BraveLocationBarModelDelegate(
     Browser* browser) : BrowserLocationBarModelDelegate(browser) {}
 
 BraveLocationBarModelDelegate::~BraveLocationBarModelDelegate() {}
 
-base::string16
-BraveLocationBarModelDelegate::FormattedStringWithEquivalentMeaning(
-    const GURL& url,
-    const base::string16& formatted_url) const {
-  base::string16 new_formatted_url =
-      BrowserLocationBarModelDelegate::FormattedStringWithEquivalentMeaning(
-          url, formatted_url);
-
+// static
+void BraveLocationBarModelDelegate::FormattedStringFromURL(const GURL& url,
+    base::string16* new_formatted_url) {
   if (url.SchemeIs("chrome")) {
     base::ReplaceFirstSubstringAfterOffset(
-        &new_formatted_url,
+        new_formatted_url,
         0,
         base::UTF8ToUTF16("chrome://"),
         base::UTF8ToUTF16("brave://"));
   }
 
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
```

Comment:
same as above BRAVE_WALLET_ENABLED -> ENABLE_EXTENSIONS

Reply:
sorry, this one should still be `BRAVE_WALLET_ENABLED`

Reply:
Confirming no change.